src/ext/java/org/opentripplanner/ext/ridehailing/service/uber/UberTripTimeEstimateResponse.java

```diff
@@ -0,0 +1,15 @@
+package org.opentripplanner.ext.ridehailing.service.uber;
+
+import java.util.List;
+
+public record UberTripTimeEstimateResponse(List<UberTripTimeEstimate> prices) {
+  public static class UberTripTimeEstimate {
```

Comment:
`public static` innerclass is not allowed by over coding guideline.

Reply:
I neglected to convert this to a record. Done in https://github.com/opentripplanner/OpenTripPlanner/pull/4979/commits/af5dd5e9325286846ccf72d4f9f211ad3a3d6f3e